sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -134,10 +135,14 @@ private void startTracing(final @NonNull Activity activity) {
 
       // in case appStartTime isn't available, we don't create a span for it.
       if (firstActivityCreated || appStartTime == null) {
-        transaction = hub.startTransaction(activityName, UI_LOAD_OP);
+        transaction =
+            hub.startTransaction(
+                new TransactionContext(activityName, UI_LOAD_OP), null, false, null, true);
       } else {
         // start transaction with app start timestamp
-        transaction = hub.startTransaction(activityName, UI_LOAD_OP, appStartTime);
+        transaction =
+            hub.startTransaction(
+                new TransactionContext(activityName, UI_LOAD_OP), null, false, appStartTime, true);
```

Comment:
@maciejwalkowiak I am branched from your branch `gh-1523`, can we make an easy overload for such option? that allows name, op and waitForChildren, with or without `startTimestamp`

Reply:
Sorry for late reply. This overload is already merged to master https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/IHub.java#L387